markets/bob/index.ts

```diff
@@ -0,0 +1,50 @@
+import { eBobNetwork, IAaveConfiguration } from "./../../helpers/types";
+import AaveMarket from "../aave";
+import {
+  strategyDAI,
+  strategyUSDC,
+  strategyWETH,
+} from "./reservesConfig";
+import { ZERO_ADDRESS } from "../../helpers";
+
+export const BobConfig: IAaveConfiguration = {
+  ...AaveMarket,
+  TestnetMarket: false,
+  ProviderId: 111,
+  MarketId: "Build on Bitcoin",
+  WrappedNativeTokenSymbol: "WETH",
+  ReservesConfig: {
+    DAI: strategyDAI,
+    USDC: strategyUSDC,
+    WETH: strategyWETH,
+  },
+  ReserveFactorTreasuryAddress: {
+    // [eBobNetwork.testnet]: "0xb7ba1Dea4a3745e58959a2091b47096cc197be5A", // config just mainnet. testnet is deployer (01_treasury.ts)
+  },
+  ParaswapRegistry: {
+    // [eBobNetwork.testnet]: "0xb7ba1Dea4a3745e58959a2091b47096cc197be5A",
```

Comment:
Maybe this is what I was mentioning the other day. I think is for repayment with collateral but we can give it a try...